### DISCUSSION TITLE
Sandbox for React player?

### DISCUSSION THREAD

**Discussion by pietrop** [vidstack/player#696] [Q&A]
Love the 💡 of the Sandbox for the player. Is there one for react?

_Happy to contribute with a PR if that helps_

**Comment by mihar-22** [comment]
Ye we should definitely have a sandbox for React. I'm kind of contemplating whether we should add Storybook or not. I'll come back with more answers when I finish migration work.

**Reply by pietrop** [reply]
ok, cool, sounds good. 

Yeah, it's interesting how to draw the line between the user facing docs for using it as "a black box" 📦  and the docs for the "internal components"... will circle back if I get any 💡
